src/main/java/com/powsybl/openloadflow/ac/nr/NewtonRaphson.java

```diff
@@ -64,8 +63,8 @@ public static List<Pair<Equation<AcVariableType, AcEquationType>, Double>> findL
                 .collect(Collectors.toList());
     }
 
-    private NewtonRaphsonStatus runIteration(StateVectorScaling svScaling) {
-        LOGGER.debug("Start iteration {}", iteration);
+    private NewtonRaphsonStatus runIteration(StateVectorScaling svScaling, MutableInt iterations) {
+        LOGGER.debug("Start iteration {}/{}", iterations, parameters.getMaxIterations());
```

Comment:
This is weird: the max iterations is not always the total number of iterations.

Reply:
Removed